{
  "error": {
    "service": {
      "generic": "Ha ocurrido un error interno.",
      "timeout": "{{classname}}:: Error tiempo de espera en conexión: {{endpoint}}",
      "unauthorized": "No tiene permiso para acceder al recurso solicitado",
      "networkError": "Error de conexión. No ha sido posible conectar con el servidor.",
      "notFound": "Recurson no encontrado",
      "methodNotAllowed": "Método no permitido"
    }
  },
  "identification": {
    "title": "Legitimationskontroll",
    "subtitle": "Ange personnumret för den person som skall kontrolleras.",
    "send": "Skicka",
    "message": "Personnummer"
  },
  "message": {
    "title": "leg-kontroll slutförd",
    "message": "Processen för leg-kontroll har slutförts",
    "start": "Starta en ny leg-kontroll"
  },
  "back": {
    "title": "Pantalla error",
    "msg": "Pantalla de cancelar al pulsa volver"
  },
  "scanner": {
    "error": "Ha ocurrido un error"
  }
}
